social icons added and cards ui enhanced

Bhavika-Gianey opened the pull request:
## Please, go through these steps before you submit a PR.
fixes #11 

## Issue Description 
Enhanced the card UI and added the social icon buttons in the Meet The Pod team .

## Before

![o1](https://user-images.githubusercontent.com/63343906/141154912-22804c26-bf61-4d1c-9c73-5ea48c90be21.PNG)


## After

![Screenshot (5)](https://user-images.githubusercontent.com/63343906/141154966-c92e5e56-6a13-456e-8675-e905e1f7f6ae.png)

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

NoahCardoza commented:
Looking good! 

Generally, it's best to keep issue numbers out of the PR title. Also, if you prefix the issue number in the body of the PR with "fix" or "fixes" you'll it will automatically link it to the issue, and when the PR is merged it will close the issue. How cool is that?!

NoahCardoza commented on _sass/description.scss:
```diff
@@ -0,0 +1,64 @@
+body {
+    // background-image: linear-gradient(#3F51B5, #1A237E);
```
If you don't need this I'd remove it. It's usually not a good idea to push commented code.

NoahCardoza commented on _includes/description.html:
```diff
@@ -1,11 +1,19 @@
+        <h3 class="name">{{ item.name }}</h3>
+        <p class="title">{{ item.location }}</p>
+        <div class="social">
+          <a href={{ item.linkedin }}><i class="fa fa-linkedin-in"></i></a>
```
I think it might be best to conditionally render the icons only if they have respective links to be clicked. Otherwise, users will be clicking the icons and thinking the site is broken 😱

Bhavika-Gianey replied:
Noted, will make the changes.

NoahCardoza commented:
I'm thinking it would be best to not add any data in this PR but rather collaborate with @shravani05  and @Pratyush1606 since they are already collection data for #13.

Pratyush1606 commented:
@NoahCardoza we have made a draft pull request containing all the fellows' data, just waiting for 4 more images to be added. After this, we will mark the pr ready for review.

NoahCardoza commented:
Any update on this? Also, can you resolve the merge conflicts with the main branch? Thanks!

Bhavika-Gianey commented:
@NoahCardoza , I have made the changes , added the conditions on social icons ,reverted the gemfile.lock file and resolved the conflicts.
Please review the changes .

NoahCardoza commented:
Looks good 🚀
